builder-22.toml

```diff
@@ -0,0 +1,31 @@
+description = "Base builder for Heroku-22 stack, based on ubuntu:22.04 base image"
+
+[stack]
+id = "heroku-22"
+build-image = "heroku/pack:22-build"
+run-image = "heroku/pack:22"
+
+[lifecycle]
+version = "0.13.5"
+
+[[buildpacks]]
+  id = "heroku/procfile"
+  uri = "docker://docker.io/heroku/procfile-cnb:1.0.1"
+
+[[buildpacks]]
+  id = "heroku/nodejs"
+  uri = "docker://public.ecr.aws/heroku-buildpacks/heroku-nodejs-buildpack@sha256:c4de76730ea8d0c68ccf1c073734ae467890545eec5896fa5ae805e2883ff20d"
+
+[[buildpacks]]
+  id = "heroku/nodejs-function"
+  uri = "docker://public.ecr.aws/heroku-buildpacks/heroku-nodejs-function-buildpack@sha256:6989739efd5c866907bedf235e0e350bb53938d482377c1bf116f70752794f41"
+
+[[order]]
+  [[order.group]]
+    id = "heroku/nodejs-function"
+    version = "0.9.1"
```

Comment:
This buildpack version and the one below differs from that in the heroku-20 builder. That is quite possibly expected (since the most recent changes to the nodejs CNB were just to add Heroku-22 support, which the other builders don't need), but mentioning just in case it wasn't :-)

Reply:
Yes, this was intentional. I wasn't sure how long this branch would hang around, so was avoiding updating `builder-18.toml` and `builder-20.toml` to prevent potential merge conflicts later.